data.sql

```diff
@@ -31,6 +31,40 @@ INSERT INTO species (name) VALUES
 ('Digimon');
 
 
+INSERT INTO vets (name,age, date_of_graduation) VALUES 
+('William Tatcher',45,'2000-04-23'),
+('Maisy Smith',26,'2019-01-17'),
+('Stephanie Mendez',64,'1981-05-4'),
+('Jack Harkness',45,'2008-06-08');
+
+INSERT INTO specializations (vid,sid) VALUES 
+(1,1),
+(3,1),
+(3,2),
+(4,2);
+
+INSERT INTO visits (aid,vid,date_of_visits) VALUES 
+(23,1,'2020-05-24'),
```

Comment:
[OPTIONAL ] 
- Rather than hardcoded data, you could also use `INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');`